src/sidebar/components/Annotation/AnnotationBody.tsx

```diff
@@ -80,10 +81,14 @@ function AnnotationBody({ annotation, settings }: AnnotationBodyProps) {
 
   const textStyle = applyTheme(['annotationFontFamily'], settings);
 
-  const shouldLinkTags = useMemo(
-    () => annotation && !isThirdPartyUser(annotation?.user, defaultAuthority),
+  const authorIsThirdParty = useMemo(
+    () => isThirdPartyUser(annotation.user, defaultAuthority),
     [annotation, defaultAuthority],
   );
+  const mentionMode = useMemo(
+    (): MentionMode => (authorIsThirdParty ? 'display-name' : 'username'),
+    [authorIsThirdParty],
+  );
```

Comment:
I just realized this could just be `const mentionMode: MentionMode = authorIsThirdParty ? 'display-name' : 'username'`. The `useMemo` overhead is not worth it here.